src/shared/components/ActionBar/ActionBar.style.ts

```diff
@@ -78,7 +78,12 @@ export const DetailsContainer = styled.div`
   margin-left: auto;
   display: flex;
   align-items: center;
-  padding: 0 ${sizes(5)};
+  padding: 13px ${sizes(4)};
+  transition: background-color ${transitions.timings.sharp} ${transitions.easing};
+
+  :hover {
+    background-color: ${colors.transparentPrimary[18]};
+  }
```

Comment:
Now that this element has a hover state, I think we should also make the text non-selectable so you don't get a text selection cursor when hovering over text